draft-ietf-ice-pac.xml

```diff
@@ -238,6 +238,32 @@
         pairs prior to the timer elapsing.
       </t>
     </section>
+    
+    <section title="Update to Trickle ICE">
+      <t>
+        Trickle ICE <xref target="I-D.ietf-ice-trickle">
+        considers a similar problem, namely whether an ICE agent should allow
+        a checklist to enter the Failed state if more candidates might
+        still be provided by the remote peer. The solution, specified in 
+        <xref target="I-D.ietf-ice-trickle">, Section 8, is to
+        wait until an end-of-candidates indication has been received
+        before determining ICE failure.
+      </t>
+      <t>
+        However, the end-of-candidates indication may not include the
+        discovery of peer-reflexive candidates, and so the solution
```

Comment:
It sounds strange to say that the indication does not include the discovery. I suggest something like:

"However, an ICE agent may receive peer-reflexive candidates after it has received the end-of-candidates indication, and so the solution...."

In addition, I think we should say that if Tpac expires before the end-of-candidates has been received, the ICE agent will continue with normal trickle procedures.

Reply:
Good suggestion, updated accordingly. I didn't include anything about PAC firing before EOC yet because there will be cases where no EOC is received.